lib/chef/application/windows_service.rb

```diff
@@ -264,6 +266,14 @@ def auto_log_level?
         Chef::Config[:log_level] == :auto
       end
 
+      def resolve_log_location
+        if Chef::Config[:log_location] && Chef::Config[:log_location].is_a?(String)
```

Comment:
yeah I explicitly remember starting to add the conditional and then removing it last night but I think keeping it is the right thing. I'll add that.